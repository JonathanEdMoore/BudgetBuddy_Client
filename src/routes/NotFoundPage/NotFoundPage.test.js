import React from 'react'
import ReactDOM from 'react-dom'
import NotFoundPage from './NotFoundPage'
import { BrowserRouter } from 'react-router-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('NotFoundPage smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><NotFoundPage /></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

describe('NotFoundPage snapshot test', () => {
  it('renders the UI as expected', () => {
    const wrapper = shallow(<NotFoundPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})