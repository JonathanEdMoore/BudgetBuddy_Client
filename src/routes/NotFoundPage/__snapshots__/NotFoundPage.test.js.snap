// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage snapshot test renders the UI as expected 1`] = `
<div>
  <MainHeader />
  <section>
    <header>
      <h3>
        Sorry! Page Not Found.
      </h3>
    </header>
  </section>
</div>
`;
